src/server/cluster/cluster_data.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <string_view>
+#include <unordered_set>
+#include <vector>
+
+#include "src/core/fibers.h"
+
+namespace dfly {
+
+class ClusterData {
```

Comment:
This isn't really data, right? As in, this class is not responsible for any data nor data handling.
Maybe we can call it `ClusterSlots`? Or `ClusterKeying`?

Reply:
So how about `ClusterConfiguration`?
I personally think that "data" is too generic, but obviously your call.

Reply:
So how about `ClusterConfiguration`?
I personally think that "data" is too generic, but obviously your call.

Reply:
ok will change